composer.json

```diff
@@ -7,11 +7,11 @@
     "require": {
         "php": "^8.1",
         "silverstripe/recipe-plugin": "2.x-dev",
-        "silverstripe/recipe-cms": "5.x-dev",
-        "silverstripe/documentconverter": "3.x-dev",
-        "silverstripe/iframe": "3.x-dev",
-        "silverstripe/tagfield": "3.x-dev",
-        "silverstripe/taxonomy": "3.x-dev"
+        "silverstripe/recipe-cms": "6.x-dev",
+        "silverstripe/documentconverter": "4.x-dev",
```

Comment:
There's no way we're making document converter CMS 6 compatible, so remove that.

Reply:
We'll create a CMS 6 branch for now and look at support at a later date

Reply:
That's ridiculous but sure.